docs/core/rid-catalog.md

```diff
@@ -161,32 +175,32 @@ See [Prerequisites for .NET Core on Windows](windows-prerequisites.md) for more
   - `linuxmint.18.1-x64` (.NET Core 2.0 or later versions)
   - `linuxmint.18.2-x64` (.NET Core 2.0 or later versions)
   - `linuxmint.18.3-x64` (.NET Core 2.0 or later versions)
-- SUSE Enterprise Linux (SLES) (.NET Core 2.0 or later versions)
-  - `sles-x64`
-  - `sles.12-x64`
-  - `sles.12.1-x64`
-  - `sles.12.2-x64`
-  - `sles.12.3-x64`
-- Alpine Linux (.NET Core 2.1 or later versions)
-  - `alpine-x64`
-  - `alpine.3.7-x64`
 
 See [Prerequisites for .NET Core on Linux](linux-prerequisites.md) for more information.
 
 ## macOS RIDs
 
-macOS RIDs use the older "OSX" branding.
-
-- `osx-x64` (.NET Core 2.0 or later versions, minimum version is `osx.10.12-x64`)
-- `osx.10.10-x64`
-- `osx.10.11-x64`
-- `osx.10.12-x64` (.NET Core 1.1 or later versions)
-- `osx.10.13-x64`
+macOS RIDs use the older "OSX" branding. Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
+
+- Portable (.NET Core 2.0 or later versions)
+  - `osx-x64` (Minimum OS version is `osx.10.12-x64`)
+- macOS 10.10  Yosemite
+  - `osx.10.10-x64`
+- macOS 10.11 El Capitan
+  - `osx.10.11-x64`
+- macOS 10.12 Sierra (.NET Core 1.1 or later versions)
+  - `osx.10.12-x64`
+- macOS 10.13 High Sierra (.NET Core 1.1 or later versions)
+  - `osx.10.13-x64`
+- macOS 10.14 Mojave (.NET Core 1.1 or later versions)
+  - `osx.10.14-x64`
 
 See [Prerequisites for .NET Core on macOS](macos-prerequisites.md) for more information.
 
 ## Android RIDs (.NET Core 2.0 or later versions)
 
+Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
```

Comment:
I would delete the Android section here. There are no official/supported Android packages available today.